ZipBuilder/Sources/ZipBuilder/CocoaPod.swift

```diff
@@ -124,8 +126,39 @@ public enum CocoaPod: String, CaseIterable {
               "GoogleMobileVision.framework",
               "LabelDetector.framework",
               "Protobuf.framework"]
-    default:
-      // By default, no folders need to be removed.
+    case .mlVisionAutoML:
+      return ["BarcodeDetector.framework",
```

Comment:
Does it make sense to make this automatic and include all frameworks in the pod? Perhaps a future enhancement?

Reply:
Updated the TODO about automating this to looking at `vendored_frameworks`. Thanks for the suggestion!